dev/index.d.ts

```diff
@@ -109,42 +114,42 @@ export declare function gulp_webpack(mode: string): cp.ChildProcess;
  * Loose type to use for T1 and T2 versions of "@azure/ms-rest-js".  The Azure Account extension returns
  * credentials that will satisfy both T1 and T2 requirements
  */
- export type AzExtServiceClientCredentials = AzExtServiceClientCredentialsT1 & AzExtServiceClientCredentialsT2;
```

Comment:
what happened here

Reply:
Dude... I have no idea =_=

Reply:
I think that whenever this file was edited (most likely copied/pasted) a stray prefixed space snuck in. I think how it is now is properly formatted.

![image](https://user-images.githubusercontent.com/5290572/213547511-54ed77a7-be6f-42f5-8094-36f2d47cf2b1.png)